packages/dictionary/package.json

```diff
@@ -1,6 +1,6 @@
 {
-	"name": "dictionary",
-	"version": "1.0.0",
+	"name": "@overture-stack/lectern-dictionary",
```

Comment:
This will be a publicly available dependency of both the client and server. Now set to be published under `@overture-stack` organization.